model_compression_toolkit/core/common/quantization/quantize_graph_weights.py

```diff
@@ -23,20 +23,21 @@
 from model_compression_toolkit.logger import Logger
 
 
-def quantize_graph_weights(graph: Graph) -> Graph:
+def quantize_graph_weights(graph_to_quantize: Graph) -> Graph:
```

Comment:
maybe this is for a separate PR, but I don't like this function, especially since it is under common/quantization.
It can be mistakenly considered as THE function for quantizing the graph, where in practice it is only used for the analyzer (and for second-moment correction, which I think is by mistake and also is used nowhere).

To sum up - I think we should move this method to be under the analyzer package, give it proper documentation for its use, and fix the second moment so it won't use it.
If you agree then consider if you want to do this in this PR or in a separate one.

Reply:
Since second-moment also depends on it, we should think about it in a separate PR.